src/_http/http2Adapter.js

```diff
@@ -36,29 +44,68 @@ function Http2Adapter() {
  * @param {?boolean} isStreaming Whether it's a streaming request. A separate session
  * is created for streaming requests to avoid shared resources with regular
  * ones for the purpose of reliability.
- * @returns {ClientHttp2Session} Http2 session.
+ * @returns {object} Http2 session API.
```

Comment:
nit: I'm not a huge fan of "session API" as a name for this object. "API" makes me think of something exterior-facing.

Maybe we can call it a "sessionAdapter" instead (to match the Http2Adapter)?

Reply:
@retwere Good catch, it's confusing. I've named it API as it's supposed to be like an interface to work with a session. How do you feel about "sessionInterface"?

Reply:
Sounds good to me 👍